tests/py2puml/inspection/test_inspectclass.py

```diff
@@ -0,0 +1,69 @@
+
+from importlib import import_module
+
+from py2puml.domain.umlclass import UmlClass, UmlAttribute
+from py2puml.domain.umlrelation import UmlRelation, RelType
+from py2puml.inspection.inspectmodule import inspect_module
+from py2puml.parsing.astvisitors import ConstructorVisitor
+from py2puml.parsing.moduleresolver import ModuleResolver
+from py2puml.parsing.parseclassconstructor import parse_class_constructor
+
+from tests.asserts.attribute import assert_attribute
+from tests.asserts.relation import assert_relation
```

Comment:
I am afraid that such a file would grow to big with time. However, I could add a `test/asserts.__init__.py` where all these assertion functions could be imported from, wdyt?

Reply:
> I am afraid that such a file would grow to big with time.

As of now, some files consist of solely a few lines. They can always be split then if growing too big.

Alternatively, I you want to stick to several files, adding a `__init__.py` for `tests.asserts` is probably suitable. This would also allowing avoiding long fully qualified module names.

Reply:
I am fine with files with only a few lines of code, they keep the balance with other bigger files :smiley: